ee/client/omnichannel/units/UnitsRoute.js

```diff
@@ -101,7 +101,7 @@ function UnitsRoute() {
 
 	const header = useMemo(() => [
 		<GenericTable.HeaderCell key={'name'} direction={sort[1]} active={sort[0] === 'name'} onClick={onHeaderClick} sort='name' w='x120'>{t('Name')}</GenericTable.HeaderCell>,
-		<GenericTable.HeaderCell key={'visibility'} direction={sort[1]} active={sort[0] === 'visibility'} onClick={onHeaderClick} sort='visibility' w='x200'>{t('Visibility')}</GenericTable.HeaderCell>,
+		<GenericTable.HeaderCell key={'visibility'} direction={sort[1]} active={sort[0] === 'visibility'} onClick={onHeaderClick} sort='visibility' w='x60'>{t('Visibility')}</GenericTable.HeaderCell>,
```

Comment:
The remove button isn't aligned properly anymore.

![Screen Shot 2020-11-18 at 10 12 39](https://user-images.githubusercontent.com/59577424/99534737-a6756500-2986-11eb-9a83-bd55bfbbed5e.png)

Reply:
@rafaelblink I'm still facing the same issue. The changes you submitted didn't solve the issue.

![Screen Shot 2020-11-18 at 13 56 36](https://user-images.githubusercontent.com/59577424/99561740-0b8c8300-29a6-11eb-9d48-9d710f1cfde6.png)

Reply:
This is because the width of columns needs to be auto. I'm pushing a possible fix.